### ISSUE TITLE
Town Reviewer added when not in a town 

### ISSUE THREAD

**Issue by ABonomo21** [dhenry-KCI/FredCo-Post-Go-Live-#432]
The town reviewer for Brunswick generated on permit 265180 when the property is not located in the town limits, 

address is 6900 ENGLISH MUFFIN WAY SUITE M-P FREDERICK MD 21703
parcel is 01021524

town ship code is 00
![image](https://user-images.githubusercontent.com/47611580/74958039-60c74400-53d6-11ea-9cbe-bb4c2bd00841.png)

staff waived but this should not have occurred.

**Comment by calequin** [comment]
the township code on this address is listed as 01 = Brunswick
![image](https://user-images.githubusercontent.com/47603568/74965975-d681de80-53db-11ea-95e7-1dce95ad9a34.png)

**Comment by JuanitaMiles** [comment]
@calequin I believe the accurate township codes are held on the Parcel detail rather than the address. If you open the linked parcel to this address the Township code shows as 00.

![image](https://user-images.githubusercontent.com/31770504/74966623-64ae9280-53e5-11ea-89a6-873bdbe08e7c.png)

**Comment by calequin** [comment]
@JuanitaMiles GIS will need to make sure that parcel township and address township codes are matching. Building permits workflows rely heavily on the address field and changing it to the parcel will require a mass amount of rework.

**Comment by JuanitaMiles** [comment]
Sent an email to the project and GIS team requesting some information on this. 2/21/20 11:50am. Awaiting response.

**Comment by JuanitaMiles** [comment]
I've replicated this application in TEST (263386) and tried it with the updated address township code. The town did not add as an applicant to the application. This correction was also made by Mary in the Prod environment. I've created high priority ticket #435 to address the overall issue with some of the addresses not having a correct township code. Closing this ticket.

![image](https://user-images.githubusercontent.com/31770504/75203490-32ca6280-573c-11ea-8abf-81e13b42fb53.png)
